website/src/data/buttons.json

```diff
@@ -851,5 +851,63 @@
                 "usesColors": false
             }
         ]
+    },
+    {
+        "type": "multistate",
+        "label": "Multistate Buttons",
+        "classes": "multi-state-btn",
+        "shouldHaveRoundedType": true,
+        "shouldHaveBlockType": false,
+        "variations": [
+            {
+                "label": "2 States Button",
+                "classes": "states-2-btn"
+            },
+            {
+                "label": "3 States Button",
+                "classes": "states-3-btn"
+            },
+            {
+                "label": "4 States Button",
+                "classes": "states-4-btn"
+            },
+            {
+                "label": "5 States Button",
+                "classes": "states-5-btn"
+            },
+            {
+                "label": "6 States Button",
+                "classes": "states-6-btn"
+            }
+        ],
+        "children": [
+            {
+                "label": "State Button",
+                "type": "state-btn",
+                "classes": "state-btn",
+                "variations": [
+                    {
+                        "label": "2 States Button",
+                        "html": "{button context}<span>{button state content here}<span/><span>{button state content here}<span/>"
+                    },

```

Comment:
What are these for? The html and label and all of that. I'm kind of confused on what they're for.

I suggest looking at the data for the basic buttons I think it might be similar to your use case.

Reply:
the button should look like this

<button class="multi-state-btn states-2-btn">
     {an icon or text here}
     <span class="state-btn">{an icon or text here}</span>
     <span class="state-btn">{an icon or text here}</span>
<button/>

I didn't know exactly how to convert this into the buttons.json